src/masternodes/loan.h

```diff
@@ -122,6 +122,7 @@ using CVaultId = uint256;
 struct CVaultMessage {
     std::string ownerAddress;
     std::string schemeId;
+    bool isLiquidated{false};
```

Comment:
`isLiquidated` is not a one-way switch. A vault can be under liquidation, and once successfully recovered its assets and get above minimum collateralization ratio, it can exit liquidation state. 

Probably `bool isUnderLiquidation` is a better name.